### PR TITLE
rapidjson: expose declared dependency from the wrap

### PR DESCRIPTION
**Pull request by dvrogozh**
Fixes: #1084

### CODE REVIEW

File: subprojects/rapidjson.wrap [hunk 1]
```diff
@@ -4,3 +4,7 @@ source_url = https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz
 source_filename = rapidjson-1.1.0.tar.gz
 source_hash = bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
 patch_directory = rapidjson
+
+[provide]
+rapidjson = rapidjson_dep
+
```
Comment (neheb) [inline subprojects/rapidjson.wrap:10]: extra newline not needed and recommended against.

File: releases.json [hunk 2]
```diff
@@ -2247,7 +2247,11 @@
     ]
   },
   "rapidjson": {
+    "dependency_names": [
+      "rapidjson",
```
Comment (neheb) [inline releases.json:2251]: get rid of the comma.

File: subprojects/rapidjson.wrap (hunk 1 above)
Comment (dvrogozh) [reply subprojects/rapidjson.wrap]: done

File: releases.json (hunk 2 above)
Comment (dvrogozh) [reply releases.json]: done

**Pull request merged**